docker-entrypoint.sh

```diff
@@ -29,7 +29,8 @@ fi
 if [[ ! -z "$@" ]]; then
     "$@"
 elif [[ "$DEV_SERVER" = "1" ]]; then
-    python ./manage.py runserver 0.0.0.0:8082
+    ./manage.py runserver 0.0.0.0:8082
 else
+    ./manage.py import_features
```

Comment:
Should we also import features in dev mode?

Reply:
We can, I'll refactor and add an environment variable to configure that behaviour.